### ISSUE TITLE
Group and light do not respond to off command

### ISSUE THREAD

**Issue by DotOnedotNL** [Foddy/node-red-contrib-huemagic#135]
**Describe the bug**
When sending an "off" command to a light or a group, the lights do not respond correctly and stay on.

**send**: { "on": true }
**reply**: {"on":true,"allOn":true,"anyOn":true,"brightness":50,"rgb":[255,217,180],"hex":"ffd9b4","colorTemp":266,"updated":"2020-01-21T14:18:43+01:00"}

**send**: { "on": false }
**reply**: {"on":true,"allOn":true,"anyOn":true,"brightness":50,"rgb":[255,217,180],"hex":"ffd9b4","colorTemp":266,"updated":"2020-01-21T14:28:36+01:00"}

Also when setting brightness to 0, the lights stay on.

**send**: { "brightness": 50 }
**reply**: {"on":true,"allOn":true,"anyOn":true,"brightness":50,"rgb":[255,217,180],"hex":"ffd9b4","colorTemp":266,"updated":"2020-01-21T14:30:25+01:00"}

**send**: { "brightness": 50 }
**reply**: {"on":false,"allOn":true,"anyOn":true,"brightness":50,"rgb":[255,217,180],"hex":"ffd9b4","colorTemp":266,"updated":"2020-01-21T14:31:34+01:00"}

**Flow to Reproduce**
```
[{"id":"1fe6840b.082eb4","type":"debug","z":"890c3864.b09828","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":750,"y":120,"wires":[]},{"id":"9a8f56ff.00c8f","type":"hue-group","z":"890c3864.b09828","name":"Woonkamer","bridge":"f62ef2f8.e34d68","groupid":"1","colornamer":false,"skipevents":false,"x":590,"y":120,"wires":[["1fe6840b.082eb4"]]},{"id":"896c4e46.b09608","type":"inject","z":"890c3864.b09828","name":"","topic":"","payload":"{ \"brightness\": 0 }","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":420,"y":120,"wires":[["9a8f56ff.00c8f"]]},{"id":"30f30c9a.e10624","type":"inject","z":"890c3864.b09828","name":"","topic":"","payload":"{ \"brightness\": 50 }","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":410,"y":180,"wires":[["9a8f56ff.00c8f"]]},{"id":"c01f77b1.ca973","type":"debug","z":"890c3864.b09828","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":750,"y":340,"wires":[]},{"id":"5de575d5.187d1c","type":"hue-group","z":"890c3864.b09828","name":"Woonkamer","bridge":"f62ef2f8.e34d68","groupid":"1","colornamer":false,"skipevents":false,"x":590,"y":340,"wires":[["c01f77b1.ca973"]]},{"id":"3b5e6335.db241c","type":"inject","z":"890c3864.b09828","name":"","topic":"","payload":"{ \"on\": true }","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":410,"y":340,"wires":[["5de575d5.187d1c"]]},{"id":"afd91cbd.ed9ae","type":"inject","z":"890c3864.b09828","name":"","topic":"","payload":"{ \"on\": false }","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":410,"y":400,"wires":[["5de575d5.187d1c"]]},{"id":"f62ef2f8.e34d68","type":"hue-bridge","z":"","name":"Philips hue","bridge":"192.168.1.42","key":"hNhtLTnAsGpPxX8te7T2YkDCtJSCXwuq5R4KIKzA","interval":"3000","disableupdates":false}]
```

**Expected behavior**
that's obvious, when sending the "on": false the light should go off. Similarly when setting brightness to 0

**Screenshots**
<img width="1154" alt="Screenshot 2020-01-21 at 14 36 05" src="https://user-images.githubusercontent.com/19171666/72809181-6a388180-3c5b-11ea-8b27-7b19ad47cf11.png">
<img width="1152" alt="Screenshot 2020-01-21 at 14 35 51" src="https://user-images.githubusercontent.com/19171666/72809183-6c9adb80-3c5b-11ea-86c3-783e40a290a3.png">

**Please complete the following information:**
 - NodeRED version: 0.18.7
 - NodeJS version: 10.18.1
 - Device running NodeRED: ubuntu 18.0.4 bare metal install

**Additional context**
All worked fine in previous releases

**Comment by dongennl** [comment]
Having similar issues; think this is still related to issue #131 ; here's what I posted trying to re-open that one:
---
I'm afraid I'm still having issues with 2.5.2
(running on Docker, restarted container multiple times after upgrade; using 'real' Hue)

This is still **ignored** by the hue-light node (though the error message is gone). The light does not turn off.
msg.payload = { transitionTime: 20, on: false }; return msg;

This **does** work:
msg.payload = false; return msg;

Not sure what is causing all this when it worked before; have had to reprogram 5 rooms as lights all stayed on!

**Comment by pcmvanveen** [comment]
Same problem here. Temp fix is working fine for me

return {payload:false};

**Comment by DotOnedotNL** [comment]
indeed, at least a work-around.
replaced hue-light.js and hue-group.js

**Comment by Foddy** [comment]
Sorry! The error has just been fixed and should no longer occur with v2.5.3!
